lib/cli/commands/closechannel.ts

```diff
@@ -0,0 +1,30 @@
+import { Arguments } from 'yargs';
+import { CloseChannelRequest } from '../../proto/xudrpc_pb';
+import { callback, loadXudClient } from '../command';
+
+export const command = 'closechannel <node_identifier> <currency>';
```

Comment:
This was actually intentional since it's a flag, I'm envisioning it's usage as just adding `--force` to the command. 

It looks like I can put `closechannel <node_identifier> <currency> [--force]` here and then it'll work either by putting `true` at the end of the command or `--force` anywhere in the command, so that's what I'll go with.